components/log-viewer-webui/client/src/pages/SearchPage/SearchControls/search-requests.ts

```
@@ -89,10 +89,13 @@
 
     submitQuery(payload)
         .then((result) => {
-            store.updateSearchJobId(result.data.searchJobId);
-            store.updateAggregationJobId(result.data.aggregationJobId);
+            let {searchJobId, aggregationJobId}  = result.data;
```

Biome (1.9.4)

[error] This let declares some variables that are only assigned once.

'searchJobId' is never reassigned.

'aggregationJobId' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)